website/docs/about/03--accessibility-statement.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Accessibility Statement
+description: This is an accessibility statement from the HashiCorp Design Systems team.
+---
+
+Accessibility is a core requirement, not an optional feature; this aligns with our [company principles](https://www.hashicorp.com/our-principles), especially our principle of integrity.
+
+> Integrity requires a consistency of our thoughts, words, and actions and a dedication to the truth.
+
+To this end, we have built the HashiCorp Design System with accessibility as a core requirement. In this way, we are giving HashiCorp product designers and developers a greater ability to deliver accessible product experiences for all HashiCorp users.
+
+## Ensuring accessible outcomes
+
+Because we are firm in our commitment to accessibility, we take practical steps to ensure that our outcomes are accessible. This means that we integrate accessibility at every step of our creation cycle, and include team education and training as part of that. In addition to formalizing our commitment to WCAG 2.1 AA conformance with an internal policy, here are the action steps we take:
+
+- Provide education and training for our team.
+- Design with accessibility as a core requirement.
+- Conduct design reviews specifically focused on accessibility.
+- Ensure our code renders to the browser in a conformant way.
+- Use the available accessibility automation (through [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) and [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing)).
+- Manually test our code with assistive technologies.
+
+## Compatibility expectations
+
+We intend to be compatible with the following assistive technologies:
+
+- Safari with VoiceOver on macOS
+- Chrome with Jaws on Windows
+- Firefox with JAWS and NVDA on Windows
+
+## Conformance status
+
+The Web Content Accessibility Guidelines (WCAG) defines requirements for designers and developers to improve accessibility for people with disabilities. It defines three levels of conformance: Level A, Level AA, and Level AAA. The HashiCorp Design System intends to be fully conformant with WCAG 2.1 Level AA. Fully conformant means that the content fully conforms to the accessibility standard.
```

Comment:
Could we make "for designers and developers" a bit more general? It should be clear that Accessibility is really everyone's responsibility. For example, if we had a content writer on our team, there are WCAG guidelines that would also be relevant to their work.

Reply:
ok, taking a note to work on this, will do in a separate PR so we can ship the first iteration.